src/gfx.cpp

```diff
@@ -922,6 +922,21 @@ Dimension GetStringBoundingBox(StringID strid, FontSize start_fontsize)
 	return GetStringBoundingBox(buffer, start_fontsize);
 }
 
+/**
+ * Get maximum width of a list of strings.
+ * @param list List of strings, terminated with INVALID_STRING_ID.
+ * @param fontsize Font size to use.
```

Comment:
`fontsize` is unused - not sure if it needs to be used or just shouldn't be included. 🙂

Reply:
Oops, I intended to use it but forgot :/ I guess it's probably not necessary.